tmva/tmva/src/MethodDNN.cxx

```diff
@@ -1100,9 +1102,9 @@ void TMVA::MethodDNN::checkGradients ()
    fNet.addLayer (DNN::Layer (outputSize, DNN::EnumFunction::LINEAR, DNN::ModeOutputValues::SIGMOID)); 
    fNet.setErrorFunction (DNN::ModeErrorFunction::CROSSENTROPY);
    //    net.setErrorFunction (ModeErrorFunction::SUMOFSQUARES);
-
+   const int BUCKET_SIZE = 8;
```

Comment:
Hi Omar, that's because this is the checkGradients () function. As you can see above at lines 1100 - 03, this is a predefined Neural Net created just to see if the algorithm calculates gradients properly. This is independent of the actual training.